app/models.py

```diff
@@ -33,12 +33,14 @@ class User(db.Model):
                            nullable=False)
     password_changed_at = db.Column(db.DateTime, index=False, unique=False,
                                     nullable=False)
+    role = db.Column(db.String, index=False, unique=False, nullable=False)
```

Comment:
These lead to ints in the database. Would prefer strings. There is a way of doing it with python classes and postgres types, but it's quite complex so I haven't taken it on yet.